### PR TITLE
feat: US125811/add grade calculation method support.

### PR DESCRIPTION
**Pull request by mbosiljevac**
image

### CODE REVIEW

File: src/components/d2l-grade-result-presentational.js [hunk 1]
```diff
@@ -146,6 +153,15 @@ export class D2LGradeResultPresentational extends LocalizeMixin(LitElement) {
 		return html``;
 	}
 
+	_renderGradeCalculationMethod() {
+		if (this.subtitleText) {
+			return html`<div class="d2l-grade-result-presentational-subtitle d2l-body-small">
+					${this.subtitleText}
```
Comment (vix633) [inline src/components/d2l-grade-result-presentational.js:159]: does this need localization?

Comment (mbosiljevac) [reply src/components/d2l-grade-result-presentational.js:159]: This text is a property so it should be localized when it's passed to us by whoever uses the grade component

Comment (vix633) [reply src/components/d2l-grade-result-presentational.js:159]: I'm gonna check with Shepstone quick about RTL decimal numbers

File: src/components/d2l-grade-result-presentational.js [hunk 2]
```diff
@@ -146,6 +153,15 @@ export class D2LGradeResultPresentational extends LocalizeMixin(LitElement) {
 		return html``;
 	}
 
+	_renderSubtitle() {
+		if (this.subtitleText) {
+			return html`<div class="d2l-grade-result-presentational-subtitle d2l-body-small">
+					${this.subtitleText}
+				</div>
+			`;
```
Comment (JamieTran) [inline src/components/d2l-grade-result-presentational.js:161]: nit: not sure if this is needed but we might want to return empty `html` if `subtitleText` doesn't exist

**Review by JamieTran — APPROVED** [review]

File: src/components/d2l-grade-result-presentational.js (hunk 2 above)
Comment (mbosiljevac) [reply src/components/d2l-grade-result-presentational.js:161]: good point! I'll add it!

**Pull request merged**